content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -99,7 +99,7 @@ different Kubernetes components.
 | `DevicePlugins` | `true` | Beta | 1.10 | |
 | `DisableAcceleratorUsageMetrics` | `false` | Alpha | 1.19 | 1.19 |
 | `DisableAcceleratorUsageMetrics` | `true` | Beta | 1.20 | |
-| `DownwardAPIHugePages` | `false` | Alpha | 1.20 | |
+| `DownwardAPIHugePages` | `false` | Beta | 1.21 | |
```

Comment:
👋 @derekwaynecarr .
Would you leave the `Alpha` item in the table and add a value in the last column (add the last version that this feature was Alpha).
```
|`DownwardAPIHugePages` | `false` | Alpha | 1.20 | 1.20 |
```